azext_iot/central/commands_export.py

```diff
@@ -0,0 +1,136 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# Dev note - think of this as a controller
+
+from typing import List, Union
+from azext_iot.common import utility
+from azext_iot.constants import CENTRAL_ENDPOINT
+from azext_iot.central.providers import CentralExportProvider
+from azext_iot.central.models.enum import ApiVersion
+from azext_iot.central.models.v1_1_preview import ExportV1_1_preview
+
+
+def get_dataExport_export(
+    cmd,
+    app_id: str,
+    export_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+) -> Union[dict, ExportV1_1_preview]:
+    provider = CentralExportProvider(
+        cmd=cmd, app_id=app_id, api_version=api_version, token=token
+    )
+
+    return provider.get_dataExport_export(
+        export_id=export_id, central_dnx_suffix=central_dns_suffix
+    )
+
+
+def delete_dataExport_export(
+    cmd,
+    app_id: str,
+    export_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+):
+    provider = CentralExportProvider(
+        cmd=cmd, app_id=app_id, api_version=api_version, token=token
+    )
+
+    provider.delete_dataExport_export(
+        export_id=export_id, central_dnx_suffix=central_dns_suffix
+    )
+
+
+def list_dataExport_exports(
+    cmd,
+    app_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+) -> List[Union[dict, ExportV1_1_preview]]:
+    provider = CentralExportProvider(
+        cmd=cmd, app_id=app_id, api_version=api_version, token=token
+    )
+
+    return provider.list_dataExport_exports(central_dns_suffix=central_dns_suffix)
+
+
+def add_dataExport_export(
+    cmd,
+    app_id: str,
+    export_id: str,
+    source,
+    destinations,
+    display_name,
+    enabled=True,
+    filter=None,
+    enrichments=None,
```

Comment:
Similar questions here as with export destination - are these all the possible options?

Reply:
yes